.github/workflows/canary.yml

```diff
@@ -1,7 +1,8 @@
 name: Canary (published artifacts) build
 
 on:
-  # TODO(anuraaga): Add cron after stabilizes
+  schedule:
+    - cron: '25 * * * *' # hourly job
```

Comment:
Weren't we going to do daily?

Reply:
want to bake canary tonight, will update it to daily tomorrow